frontend/src/components/filters/MultiSelect.tsx

```diff
@@ -9,37 +9,88 @@ import {
   MenuHandler,
   MenuChips,
 } from '../ui';
+import type { FieldGroupedOption } from '../table';
 
 export const MultiSelect = ({
   field,
   values,
   label,
   onChange,
-  handleClose,
 }: {
   field: any;
-  values: any;
+  values: string[];
   onChange: (e: ChangeEvent<HTMLInputElement>) => void;
-  handleClose: (name: string, value?: string) => void;
   label: string;
 }) => {
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Moved these functions out of the useTable hook since they are specific to this component (and to avoid passing them in from the dash since it is not shared elsewhere)